preprocessors/azure.py

```
@@ -307,26 +298,45 @@ def load_chunks(self, cfg: dict, pandas=True):
+        file_path = cfg.data_path
+        ds_store = datastore(cfg.data_store)
+
+        if 'parquet' in file_path:
+            ds = Dataset.Tabular.from_parquet_files(path=(ds_store,file_path))
+        elif ".csv" in file_path or ".asc" in file_path:
+            encodings = ['iso88591', 'utf8']
+            for encoding in encodings:
+                try:
+                    ds = Dataset.Tabular.from_delimited_files(path=(ds_store, file_path), separator=';', encoding=encoding)
+                    break
+                except (UnicodeDecodeError, VisitError):
```

Ruff (0.8.2)

Undefined name `VisitError`

(F821)